custom/icds_reports/ucr/tests/base_test.py

```diff
@@ -84,6 +84,7 @@ def _get_query_object(self):
 
     def _run_iterative_monthly_test(self, case_id, cases, start_date=date(2015, 12, 1)):
         _iteratively_build_table(self.datasource)
```

Comment:
Wanted to change this to use rebuild_indicators, but it kept failing for some reason